extension/test/fixtures/payment-no-method.json

```diff
@@ -15,6 +14,7 @@
       "key": "ctp-adyen-integration-payment-custom-type"
     },
     "fields": {
+      "interfaceId": "pspReference",
```

Comment:
I know it was like that before but `pspReference` implies that it is provided by PSP which is not the case here, right? Maybe better `paymentNoMethodReference` or just the same as in other fixtures `paymentReferenceId`?

Reply:
Have changed this to paymentReferenceId like in the rest of the tests